src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml

```diff
@@ -15,14 +15,22 @@
 		MinHeight="200"
 		Orientation="Vertical"
 		Spacing="4">
+		<muxc:InfoBar
+			Margin="0,0,0,12"
+			IsClosable="False"
+			IsIconVisible="True"
+			IsOpen="True"
+			Message="{helpers:ResourceString Name=RecentFilesDisabledOnWindowsWarning}"
+			Severity="Warning"
+			Visibility="{x:Bind IsRecentFilesDisabledInWindows, Mode=OneWay}" />
 		<TextBlock
 			x:Name="RecentItemDescription"
 			HorizontalAlignment="Stretch"
 			FontSize="14"
 			Text="{helpers:ResourceString Name=RecentItemDescription/Text}"
 			TextAlignment="Center"
 			TextWrapping="WrapWholeWords"
-			Visibility="{x:Bind Empty.Visibility, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+			Visibility="{x:Bind EmptyRecentsTextVisibility, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
```

Comment:
```suggestion
			Visibility="{x:Bind EmptyRecentsTextVisibility, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" />
```

Reply:
good catch, no clue why it was ever `TwoWay`

Reply:
Seems to be because of UpdateSourceTrigger